src/main/kotlin/com/mparticle/kits/RoktKit.kt

```diff
@@ -136,10 +132,7 @@ class RoktKit : KitIntegration(), CommerceListener, IdentityListener, RoktListen
             val widget = weakRef.get() as? Widget  // Safe cast to Widget
             widget?.let { entry.key to weakRef as WeakReference<Widget> }  // Only include if it's a Widget
         }?.toMap()
-        onUnloadCallback = onUnload
-        onLoadCallback = onLoad
-        onShouldHideLoadingIndicatorCallback = onShouldHideLoadingIndicator
-        onShouldShowLoadingIndicatorCallback = onShouldShowLoadingIndicator
+        this.mpRoktEventCallback = mpRoktEventCallback
```

Comment:
So I guess in Android you can't just pass in the code blocks like I do [here](https://github.com/mparticle-integrations/mparticle-apple-integration-rokt/commit/5b575a9e11bcd3019caebf1af0e7921a95bffa82#diff-09961b2837d7729eae6e2c8c4aec512bda7efbc0901b0b59410bd17d5fcc36ccR105)?

Reply:
We talked about this on a call. Android seems to implement this callbacks in a protocol/delegate way while iOS passes around code blocks to achieve the same thing. Interesting difference on the Rokt side to note but we're doing things correctly for now.